detect_secrets/filters/heuristic.py

```diff
@@ -165,6 +165,8 @@ def is_indirect_reference(line: str) -> bool:
 
         secret = request.headers['apikey']
     """
+    # Constrain line length as the heuristic's intention
+    # is to target lines that resemble function calls
```

Comment:
lol sorry for the back and forth but IMO this now misses the important part which is that this constraint was added to prevent the regex from taking a long time

Reply:
No worries. That's the whole point of reviews :)